reporters/cloud-run-scripts/CloudResponce.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const app = express();
+const { exec } = require("child_process");
+
+app.get("/", (req, res) => {
+  console.log("Daily reporter received a request");
```

Comment:
Do you intend to keep this `console.log` in prod?

Reply:
it logs in the GCP logs. can leave it in with the minimum cost. makes it easy to see, for example, how long the request takes to get a response. Am happy to take it out as well.

Reply:
Up to you, i'm indifferent